src/Raven.Server/Documents/PeriodicBackup/Azure/RavenAzureClient.cs

```diff
@@ -172,7 +183,7 @@ private static void GetExceptionsFromTasks(Task[] tasks)
         {
             var fillQueueTask = Task.Run(() =>
             {
-                // Put Block is limited to 4MB per block
+                // Put Block is limited to 100MB per block
```

Comment:
The idea is that there is one task that fills the queue with data and other tasks that upload it concurrently.